### DISCUSSION TITLE
How to declare scrollbar style globally?

### DISCUSSION THREAD

**Discussion by Filippinifra** [vanilla-extract-css/vanilla-extract#1199] [Q&A]
I want to style the scrollbar of my website globally, so I want that each element of my website inherit this property if is scrollable

I want to use something like that

```ts
globalStyle("*", {
  selectors: {
    "&::-webkit-scrollbar": {
      width: 50,
    },

    "&::-webkit-scrollbar-track": {
      background: "#f1f1f1",
    },

    "&::-webkit-scrollbar-thumb": {
      background: "red",
    },
  },
});
```

but this is not working and throw me an error
Someone can help me please?

**Comment by askoufis** [comment]
[The `globalStyle` API doesn't have access to `selectors`](https://vanilla-extract.style/documentation/global-api/global-style/).

You'll need to split out each selector into separate `globalStyle` calls in order to achieve your outcome:

```ts
globalStyle("*::-webkit-scrollbar", {
  width: 50,
});

globalStyle("*::-webkit-scrollbar-track", {
  background: "#f1f1f1",
});

globalStyle("*::-webkit-scrollbar-thumb", {
  background: "red",
});

```
